README.md

```diff
@@ -1,193 +1,48 @@
 # The Romana Project
 
-Romana is a new Software Defined Network (SDN) solution specifically designed
-for the Cloud Native architectural style. The result of this focus is that
-Romana cloud networks are less expensive to build, easier to operate and
-deliver higher performance than cloud networks built using alternative SDN
-designs.
+[Romana](https://romana.io) is a new [Software Defined Network (SDN)](http://romana.io/cloud/cloud_native_sdn/) solution
+specifically designed for the Cloud Native architectural style. 
+The result of this focus is that Romana cloud networks are less expensive to build, 
+easier to operate and deliver higher performance than cloud networks built using alternative SDN designs.
 
```

Comment:
I went through the README files in the repos to make this lead sentence consistent across all of them. See the branch/PR for each one. You'll see that I changed them all to be the same as the language on the project site;

'Romana is a new Software Defined Network solution specifically designed for Cloud Native applications. Romana allows multi-tenant cloud computing networks for OpenStack, Docker and Kubernetes to be built without encapsulation or a virtual network overlay.

Romana networks are less expensive to build, easier to operate and deliver higher performance than networks built using alternative overlay based SDN designs.'